src/test/scala/com/cognite/sdk/scala/v1/fdm/instances/CogniteAssetsTest.scala

```diff
@@ -0,0 +1,79 @@
+package com.cognite.sdk.scala.v1.fdm.instances
+
+import cats.effect.unsafe.implicits.global
+import com.cognite.sdk.scala.common.RetryWhile
+import com.cognite.sdk.scala.v1.fdm.Utils
+import com.cognite.sdk.scala.v1.fdm.instances.InstanceDeletionRequest.NodeDeletionRequest
+import com.cognite.sdk.scala.v1.fdm.instances.NodeOrEdgeCreate.NodeWrite
+import com.cognite.sdk.scala.v1.fdm.views.ViewReference
+import com.cognite.sdk.scala.v1.{CogniteInstanceId, CommonDataModelTestHelper, File, FileDownloadInstanceId, FileDownloadLink, FileUploadInstanceId, InstanceId}
+import sttp.client3.UriContext
+
+
+@SuppressWarnings(
+  Array(
+    "org.wartremover.warts.NonUnitStatements",
+    "org.wartremover.warts.ThreadSleep",
+    "org.wartremover.warts.Equals"
+  )
+)
+class CogniteAssetsTest extends CommonDataModelTestHelper with RetryWhile {
+
+  it should "make it possible to retrieve file and associated upload link and download link using instance id" in {
+
+    val instanceId: InstanceId = InstanceId(space = Utils.SpaceExternalId, externalId = "file_instance_ext_id2")
+    val cogniteInstanceId: CogniteInstanceId = CogniteInstanceId(InstanceId(space = Utils.SpaceExternalId, externalId = "file_instance_ext_id2"))
```

Comment:
external ids can be randomized to avoid conflicts between concurrently running tests/PRs